src/go/wsl-helper/cmd/enum.go

```diff
@@ -19,15 +19,19 @@ package cmd
 import "fmt"
 
 // enumValue describes an enumeration for use with github.com/spf13/pflag
+//
+//nolint:unused // This struct is used for the linux build and not windows
```

Comment:
The other option is to add a test and use it there. But I can understand not wanting to do that; after all, I didn't add a test when I introduced this code… :p